src/serverconf/src/main/java/ee/ria/xroad/common/conf/serverconf/dao/ServerConfDAOImpl.java

```diff
@@ -41,39 +40,43 @@
 public class ServerConfDAOImpl {
 
     /**
-     * Saves the server conf to the database.
-     * @param conf the server conf
-     * @throws Exception if an error occurs
-     */
-    public void save(final ServerConfType conf) throws Exception {
-        doInTransaction(session -> {
-            session.saveOrUpdate(conf);
-            return null;
-        });
-    }
```

Comment:
Is this method unused?

Reply:
I could not find any uses by a quick search: https://niis.slack.com/archives/GB2JH4LUE/p1573468551013000?thread_ts=1573465418.012800&cid=GB2JH4LUE